protocol-units/execution/dof/src/v1.rs

```diff
@@ -335,43 +334,37 @@ mod tests {
 			executor.execute_block_opt(block).await?;
 
 			blockheight += 1;
+			cur_ver += txns_to_commit.len() as u64;
```

Comment:
Prefer `current_version`

Reply:
Ah, nvm, this is matching by the upstream, isn't it.

Reply:
No, this is just the test code, can rename for clarity.

Reply:
https://github.com/movementlabsxyz/movement/pull/141/commits/47f348b233ce05cab970008c999b8fd1065cde5a